formbuilder/config.js

```diff
@@ -152,6 +157,10 @@ export default {
       jsonSchema: {
         type: "number",
         title: "Edit me",
+        description: "",
+        multipleOf: 1,
+        minimum: 1,
+        maximum: 100,
```

Comment:
Ah ok. I won't put 100 as a default: it seems too restrictive. Also not sure why it's a multiple of 1 by default (I would just not set this, as it lets more flexibility)